src/main/scala/se/nimsa/sbx/dicom/streams/ReverseAnonymizationFlow.scala

```diff
@@ -27,76 +31,89 @@ object ReverseAnonymizationFlow {
     Tag.ProtocolName,
     Tag.FrameOfReferenceUID)
 
-  val reverseAnonFlow = Flow[DicomPart]
+  def reverseAnonFlow: Flow[DicomPart, DicomPart, NotUsed] = Flow[DicomPart]
+    .via(groupLengthDiscardFilter)
```

Comment:
All flows that modify data (modify, anon, reverse anon, harmonize) include group length discard and to undefined length sequences flows.